src/main/kotlin/com/zepben/evolve/services/network/tracing/feeder/AssignToFeeders.kt

```diff
@@ -135,6 +136,7 @@ class AssignToFeeders {
             when (stepPath.toEquipment) {
                 is PowerTransformer -> feedersToAssign.tryEnergizeLvFeeders(stepPath.toEquipment, lvFeederStartPoints)
                 is ProtectedSwitch -> feedersToAssign.associateRelaySystems(stepPath.toEquipment)
+                is PowerElectronicsConnection -> feedersToAssign.associatePowerElectronicUnits(stepPath.toEquipment)
```

Comment:
note: With this addition you will need to update the current state processing in the network routes to remove these. This should already be happening for the other items above, so should be easy to find if you haven't already.

This also applies to the LV feeder.

Reply:
thx - done here: [zepben/ewb-network-routes@`e9c71fe` (#199)](https://github.com/zepben/ewb-network-routes/pull/199/commits/e9c71fef61613436b77479dbbb2c3697dad640ed)